include/rocksdb/wide_columns.h

```diff
@@ -133,6 +144,28 @@ class PinnableWideColumns {
   WideColumns columns_;
 };
 
+inline void PinnableWideColumns::Move(PinnableWideColumns&& other) {
+  const char* const data = other.value_.data();
+  const bool is_plain_value =
+      other.columns_.size() == 1 &&
+      other.columns_.front().name() == kDefaultWideColumnName &&
+      other.columns_.front().value() == other.value_;
```

Comment:
wondering why we need this check as part of `is_plain_value`
```
other.columns_.front().value() == other.value_;
```

Reply:
Technically, one can have wide-column entity with nothing but the anonymous default column; this is logically equivalent to a plain KV but the serialization is different under the hood and thus the column index has to be built using `CreateIndexForWideColumns`.